lua/neogit/lib/popup.lua

```diff
@@ -575,6 +575,19 @@ function M:show()
       }
     end,
   }
+
+  -- Closes the window if it loses focus and it is a floating buffer
+  if self.buffer.kind == "floating" then
+    vim.api.nvim_create_autocmd("WinLeave", {
+      callback = function(win)
+        if win.buf == self.buffer.handle then
+          -- We pcall this because it's possible the window was closed by a command invocation, e.g. "cc" for commits
+          pcall(self.close)
+          return true -- Deregisters the autocmd
```

Comment:
Is it the same as setting `once` flag?

Reply:
Huh, you're right per [the neovim source code](https://github.com/neovim/neovim/blob/33e1a8cd7042816a064c0d2bf32b6570d7e88b79/src/nvim/autocmd.c#L2083). Will change.